.pre-commit-config.yaml

```diff
@@ -43,8 +43,20 @@ repos:
   - id: black
     args: [--line-length=79]
 
-- repo: https://github.com/PyCQA/flake8
-  rev: 6.1.0
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  rev: v0.0.291
   hooks:
-  - id: flake8
-    args: [--extend-ignore=E203]
```

Comment:
This is now enforced? There does not seem any impact of this rule change just wondering why it was ignored before.